### PR TITLE
Add comment to System.gc() changes

### PR DESCRIPTION
**Pull request by tajila**

### CODE REVIEW

**Comment by tajila** [comment]
@dmitripivkine is there anything else to add?

**Comment by dmitripivkine** [comment]
> @dmitripivkine is there anything else to add?

no, I think it is ok

**Review by dmitripivkine — APPROVED** [review]

File: runtime/sunvmi/sunvmi.c [hunk 1]
```diff
@@ -624,6 +624,7 @@ JVM_GC_Impl(void)
 	vmFuncs->internalAcquireVMAccess(currentThread);
 #endif /* J9VM_INTERP_ATOMIC_FREE_JNI */
 
+	/* Two GC requests are required to guarantee a GC actually occurs */
```
Comment (keithc-ca) [inline runtime/sunvmi/sunvmi.c:627]: Please finish the sentence with a period.

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
A build is not necessary for the addition of a comment.

**Pull request merged**